xds/src/main/java/io/grpc/xds/LookasideLb.java

```diff
@@ -107,111 +127,39 @@ public void handleResolvedAddresses(ResolvedAddresses resolvedAddresses) {
     if (cfg.getError() != null) {
       throw cfg.getError().asRuntimeException();
```

Comment:
Do we really want to panic here? Instead of propagating error to channel?

Reply:
Other than throwing an exception how do you propagate error to the channel?

Reply:
Based on what grpclb is doing, it should eventually lead to a failing picker so that channel goes into transient failure with the this error propagated. A little bit involved.

Reply:
In the new service config error handling design this won't happen here. In parsing service config, the name resolver will use `LoadBalancingProvider` for the balancing policy to parse the lb config, if there is an error (`cfg.getError() != null`), it will not use this balancing policy.

If it does happen, it will be our bug, typically in test.

Reply:
Whatever the error handling mechanism used in parsing, you should only use what the API provides you and assumes no more. At least for `ConfigOrError` interface, the API designer expects you to first check if `getError() != null` before proceed to `getConfig()`, your implementation should have throughout consideration for handling the case `getError()` returns you something, not just throw a RuntimeException arbitrarily.

Reply:
After offline discussion, we agreed to goto TRANSIENT_FAILURE immediately.